queue_processors/queue_processor/reduction/service.py

```diff
@@ -176,8 +175,9 @@ def run(self, input_file, output_dir):
             return self.module.main(input_file=str(input_file.path), output_dir=str(output_dir.path))
 
 
-# We will remove the log_Stream once we look at logging in Reduction Runner more closely
-def reduce(reduction_dir, temp_dir, datafile, script):
+# pylint:disable=too-many-arguments; We will remove the log_Stream once we look at logging in ppa
```

Comment:
Is this disable still necessary?

Reply:
Strange the comment that is deleted should be the one left over.. good catch, maybe got botched in the merge